README.md

```diff
@@ -1,17 +1,25 @@
 # README
```

Comment:
Do you have a way to auto-clip the text? Otherwise, tweaking the readme will cause some gaps as it is updated. Also could introduce problems and weird text breaks on smaller screens.